fractal/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "fractal-startup",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "fractal:start": "fractal start --sync --port 3000",
+    "fractal:build": "fractal build"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@frctl/fractal": "^1.1.7"
```

Comment:
Do we want to depend on https://www.npmjs.com/package/@frctl/twig as well?

Reply:
Also, I think this should be a `devDependency` as we won't ever be running a BrowserSync powered Fractal build in production.